### DISCUSSION TITLE
Manual

### DISCUSSION THREAD

**Discussion by tbffbt** [apohl79/audiogridder#1179] [Q&A]
Where can I find the simplest yet thorough manual for AG?

I might be missing something but there doesn't seem to be anything more comprehensive the PluginTray, for example.  I've just upgraded from version 1.1.1 to 1.2.0 (and I've yet to figure out how to connect as Same Server, plugin is 'not connected') and it's the first time I am having to deal with it, yet I haven't wrapped my head around what its function is.

**Comment by apohl79** [comment]
All info i've put together is on https://audiogridder.com. 

I don't understand the problem you have. You can check the plugin and server logs. They might reveal why the connection fails.

**Reply by tbffbt** [reply]
I mean a beginner-ish level manual, which is closer to where I stand when it comes to many of the subjects one eventually comes across when dealing with AG. The function of a separate Tray plugin (considering there was none in version 1.1.1, which I've been using to this day), how to properly set up a connection, and maybe even best practices. Although I realize how it might be too much to ask, given it's an independent and in-development operation.

I figured out the connection issue has to do with LittleSnitch, it was running as usual when I first installed version 1.2.0 and it prevented Server and Plug-In from even seeing one another. After disabling LittleSnitch Filter the setup went seemingly smooth, but now, even after allowing every AudioGridder-related rule there are constant connection dropouts, which makes it impossible to use. Is there anything that's paramount I should be looking for to whitelist or disable on LittleSnitch?

I also ran into an issue upon running 1.2.0 that I hadn't before with 1.1.1 which is having to manually sync the Delay and Buffer Sizes on each instance of the plug-in to account for latency. Version 1.1.1 had automatic compensation throughout all instances, if I'm not mistaken. Is that supposed to be done all manually now?
